### PR TITLE
 Redesign transfer and home page. [WP]

### PR DESCRIPTION
**Pull request by karthikeyan-cere**

### CODE REVIEW

File: src/Components/Pages/TransferPage.tsx [hunk 1]
```diff
@@ -43,18 +43,32 @@ const showImageUrl = (url?: string) =>
 const useStyles = makeStyles(({ constants, palette }: ITheme) =>
   createStyles({
     root: {
-      padding: constants.generalUnit * 6,
+      fontFamily: "Sora, sans-serif",
+      padding: 25,
```
Comment (AndreiNavoichyk) [inline src/Components/Pages/TransferPage.tsx]: What you can not modify generalUnit coefficient only?

File: src/Components/Pages/TransferPage.tsx [hunk 2]
```diff
@@ -43,18 +43,32 @@ const showImageUrl = (url?: string) =>
...
       position: "relative",
+      backgroundColor: "#FFFFFF",
+    },
+    title: {
+      fontSize: "1.2rem",
+      fontWeight: "bold",
+      color: "#000000",
+      textAlign: "center",
+      paddingBottom: "30px",
```
Comment (AndreiNavoichyk) [inline src/Components/Pages/TransferPage.tsx]: We should use the same padding units in the new code.

File: src/Components/Pages/TransferPage.tsx [hunk 3]
```diff
@@ -43,18 +43,32 @@ const showImageUrl = (url?: string) =>
...
   createStyles({
     root: {
-      padding: constants.generalUnit * 6,
+      fontFamily: "Sora, sans-serif",
+      padding: 25,
       position: "relative",
+      backgroundColor: "#FFFFFF",
+    },
+    title: {
+      fontSize: "1.2rem",
```
Comment (AndreiNavoichyk) [inline src/Components/Pages/TransferPage.tsx]: We should use the same fontSize units in the new code.

File: src/Components/Pages/TransferPage.tsx [hunk 4]
```diff
@@ -124,16 +138,14 @@ const useStyles = makeStyles(({ constants, palette }: ITheme) =>
       borderBottomLeftRadius: 0,
       borderTopLeftRadius: 0,
       left: -1,
-      color: palette.additional["gray"][8],
-      backgroundColor: palette.additional["gray"][3],
+      color: "#5C4DCF",
```
Comment (AndreiNavoichyk) [inline src/Components/Pages/TransferPage.tsx]: It should not be copy-pasted. Reuse it.

File: src/Components/Pages/TransferPage.tsx [hunk 5]
```diff
@@ -516,19 +552,23 @@ const TransferPage = () => {
...
-            <QuestionCircleSvg
-              onClick={() => setAboutOpen(true)}
-              className={classes.faqButton}
-            />
-          </section> */}
           </Form>
         )}
       </Formik>
+      <section className={classes.footerText}>
+        <div>Cere Homepage</div>
```
Comment (AndreiNavoichyk) [inline src/Components/Pages/TransferPage.tsx]: It should be references to resources. Get them from @centilex.

File: src/Layouts/AppWrapper.tsx [hunk 6]
```diff
@@ -5,6 +5,7 @@ import { ReactNode } from "react";
 import AppHeader from "./AppHeader";
 import { ReactComponent as GlobalSvg } from "../media/Icons/global.svg";
 import { ROUTE_LINKS } from "../Components/Routes";
+import backgroundImage from "../media/background-vector-image.svg";
```
Comment (AndreiNavoichyk) [inline src/Layouts/AppWrapper.tsx]: `background-vector-image` let's rename to just `background`.

File: src/Components/Pages/TransferPage.tsx (hunk 1 above)
Comment (karthikeyan-cere) [reply src/Components/Pages/TransferPage.tsx]: Updated.

File: src/Components/Pages/TransferPage.tsx (hunk 3 above)
Comment (karthikeyan-cere) [reply src/Components/Pages/TransferPage.tsx]: Updated

File: src/Layouts/AppWrapper.tsx (hunk 6 above)
Comment (karthikeyan-cere) [reply src/Layouts/AppWrapper.tsx]: Renamed.

File: src/Components/Pages/TransferPage.tsx (hunk 4 above)
Comment (karthikeyan-cere) [reply src/Components/Pages/TransferPage.tsx]: updated

File: src/Components/Pages/TransferPage.tsx (hunk 5 above)
Comment (karthikeyan-cere) [reply src/Components/Pages/TransferPage.tsx]: Updated

File: src/Components/Pages/TransferPage.tsx (hunk 2 above)
Comment (karthikeyan-cere) [reply src/Components/Pages/TransferPage.tsx]: updated

File: src/Components/Pages/TransferPage.tsx [hunk 7]
```diff
@@ -40,21 +41,37 @@ const PredefinedIcons: any = {
...
-      padding: constants.generalUnit * 6,
+      fontFamily: "Sora, sans-serif",
+      padding: constants.generalUnit * 3,
       position: "relative",
+      backgroundColor: "#FFFFFF",
+    },
+    title: {
+      fontSize: 24,
+      fontWeight: "bold",
+      color: "#000000",
```
Comment (AndreiNavoichyk) [inline src/Components/Pages/TransferPage.tsx]: "black"?

File: src/Components/Pages/TransferPage.tsx [hunk 8]
```diff
@@ -40,21 +41,37 @@ const PredefinedIcons: any = {
...
+const textColor = "#5C4DCF";
+
 const useStyles = makeStyles(({ constants, palette }: ITheme) =>
   createStyles({
     root: {
-      padding: constants.generalUnit * 6,
+      fontFamily: "Sora, sans-serif",
+      padding: constants.generalUnit * 3,
       position: "relative",
+      backgroundColor: "#FFFFFF",
```
Comment (AndreiNavoichyk) [inline src/Components/Pages/TransferPage.tsx]: "white"?

File: src/Components/Pages/TransferPage.tsx [hunk 9]
```diff
@@ -112,6 +128,11 @@ const useStyles = makeStyles(({ constants, palette }: ITheme) =>
           borderBottomRightRadius: 0,
           borderTopRightRadius: 0,
           borderRight: 0,
+          border: "1px solid #C4C4C4",
```
Comment (AndreiNavoichyk) [inline src/Components/Pages/TransferPage.tsx]: DRY

File: src/Components/Pages/TransferPage.tsx [hunk 10]
```diff
@@ -516,19 +562,44 @@ const TransferPage = () => {
...
+          className={classes.footerText}
+          to={{ pathname: "https://cere.network/" }}
+          target="_blank"
+        >
+          Cere Homepage
+        </NavLink>
+        <NavLink
+          style={{ textDecoration: "none" }}
+          className={classes.footerText}
+          to={{ pathname: "https://explorer.cere.network" }}
```
Comment (AndreiNavoichyk) [inline src/Components/Pages/TransferPage.tsx]: Link should point to the Cere Mainnet rpc node, staking tab.

File: src/Layouts/AppWrapper.tsx [hunk 11]
```diff
@@ -39,23 +35,26 @@ const useStyles = makeStyles(
...
-        maxWidth: 460,
+        width: "100%",
         display: "flex",
+        flex: 1,
         flexDirection: "column",
         overflow: "hidden",
         borderRadius: 4,
+        alignItems: "center",
+        justifyContent: "center",
+        backgroundColor: "#F5F7F9",
```
Comment (AndreiNavoichyk) [inline src/Layouts/AppWrapper.tsx]: DRY

File: src/Components/Pages/TransferPage.tsx [hunk 12]
```diff
@@ -40,21 +41,37 @@ const PredefinedIcons: any = {
...
+      padding: constants.generalUnit * 3,
       position: "relative",
+      backgroundColor: "#FFFFFF",
+    },
+    title: {
+      fontSize: 24,
+      fontWeight: "bold",
+      color: "#000000",
+      textAlign: "center",
+      paddingBottom: 30,
```
Comment (AndreiNavoichyk) [inline src/Components/Pages/TransferPage.tsx]: Use everywhere the constants.generalUnit

File: src/Layouts/AppWrapper.tsx (hunk 11 above)
Comment (karthikeyan-cere) [reply src/Layouts/AppWrapper.tsx]: Updated

File: src/Components/Pages/TransferPage.tsx (hunk 10 above)
Comment (karthikeyan-cere) [reply src/Components/Pages/TransferPage.tsx]: Updated

File: src/Components/Pages/TransferPage.tsx (hunk 7 above)
Comment (karthikeyan-cere) [reply src/Components/Pages/TransferPage.tsx]: Updated

File: src/Components/Pages/TransferPage.tsx (hunk 8 above)
Comment (karthikeyan-cere) [reply src/Components/Pages/TransferPage.tsx]: updated

Comment (karthikeyan-cere) [reply src/Components/Pages/TransferPage.tsx]: Updated

File: src/Components/Pages/TransferPage.tsx (hunk 9 above)
Comment (karthikeyan-cere) [reply src/Components/Pages/TransferPage.tsx]: updated

File: src/Layouts/AppHeader.tsx [hunk 13]
```diff
@@ -4,29 +4,25 @@ import clsx from "clsx";
...
     return createStyles({
       root: {
+        fontFamily: "Sora, sans-serif",
         display: "flex",
-        position: "fixed",
         justifyContent: "space-between",
-        padding: `${constants.generalUnit * 2}px ${
-          constants.generalUnit * 4
-        }px`,
+        padding: "25px",
```
Comment (AndreiNavoichyk) [inline src/Layouts/AppHeader.tsx]: generalUnit

File: src/Layouts/AppHeader.tsx [hunk 14]
```diff
@@ -4,29 +4,25 @@ import clsx from "clsx";
...
         width: "100%",
-        top: 0,
-        left: 0,
-        backgroundColor: palette.additional["header"][1],
-        borderBottom: `1px solid ${palette.additional["header"][3]}`,
+        backgroundColor: "#FFFFFF",
```
Comment (AndreiNavoichyk) [inline src/Layouts/AppHeader.tsx]: black

File: src/Layouts/AppHeader.tsx [hunk 15]
```diff
@@ -75,7 +71,11 @@ const useStyles = makeStyles(
         paddingRight: 10,
       },
       title: {
-        paddingLeft: 10,
+        paddingLeft: 5,
+        fontStyle: "normal",
+        fontWeight: "bold",
+        fontSize: "30px",
```
Comment (AndreiNavoichyk) [inline src/Layouts/AppHeader.tsx]: no px

File: src/Layouts/AppHeader.tsx [hunk 16]
```diff
@@ -75,7 +71,11 @@ const useStyles = makeStyles(
         paddingRight: 10,
       },
       title: {
-        paddingLeft: 10,
+        paddingLeft: 5,
```
Comment (AndreiNavoichyk) [inline src/Layouts/AppHeader.tsx]: generalUnit

File: src/Layouts/AppHeader.tsx [hunk 17]
```diff
@@ -75,7 +71,11 @@ const useStyles = makeStyles(
...
+        fontStyle: "normal",
+        fontWeight: "bold",
+        fontSize: "30px",
+        color: "#000000",
```
Comment (AndreiNavoichyk) [inline src/Layouts/AppHeader.tsx]: black

File: src/Layouts/AppWrapper.tsx [hunk 18]
```diff
@@ -5,12 +5,15 @@ import { ReactNode } from "react";
...
 import { ReactComponent as GlobalSvg } from "../media/Icons/global.svg";
 import { ROUTE_LINKS } from "../Components/Routes";
+import backgroundImage from "../media/background.svg";
 
 interface IAppWrapper {
   children: ReactNode | ReactNode[];
   wrapTokenPage?: boolean;
 }
 
+const backgroundColour = "#F5F7F9";
```
Comment (AndreiNavoichyk) [inline src/Layouts/AppWrapper.tsx]: remove extra `u` in Colo**u**r

Comment (karthikeyan-cere) [reply src/Layouts/AppWrapper.tsx]: updated

File: src/Layouts/AppHeader.tsx (hunk 17 above)
Comment (karthikeyan-cere) [reply src/Layouts/AppHeader.tsx]: updated

File: src/Layouts/AppHeader.tsx (hunk 16 above)
Comment (karthikeyan-cere) [reply src/Layouts/AppHeader.tsx]: updated

File: src/Layouts/AppHeader.tsx (hunk 15 above)
Comment (karthikeyan-cere) [reply src/Layouts/AppHeader.tsx]: Updated

File: src/Layouts/AppHeader.tsx (hunk 14 above)
Comment (karthikeyan-cere) [reply src/Layouts/AppHeader.tsx]: Updated

File: src/Layouts/AppHeader.tsx (hunk 13 above)
Comment (karthikeyan-cere) [reply src/Layouts/AppHeader.tsx]: Updated

File: src/Components/Pages/TransferPage.tsx (hunk 12 above)
Comment (karthikeyan-cere) [reply src/Components/Pages/TransferPage.tsx]: Updated

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**